app/models/apartment.rb

```diff
@@ -0,0 +1,2 @@
+class Apartment < ApplicationRecord
```

Comment:
As part of generating a scaffold, I would also add any necessary validations or scopes to the models.